cmd/cephcsi.go

```diff
@@ -212,3 +213,11 @@ func validateCloneDepthFlag(conf *util.Config) {
 		klog.Fatalln("rbdsoftmaxclonedepth flag value should not be greater than rbdhardmaxclonedepth")
 	}
 }
+
+func validateMaxSnaphostFlag(conf *util.Config) {
+	// maximum number of snapshots on an image are 510,keeping limit to 500 to
+	// avoid issues.
```

Comment:
Is this documented as a limitation in Ceph RBD? If so, add a link to theirs docs.

/cc @dillaman

Reply:
This is the closest documentation for upstream [1][2] for 510 snapshots and 16 images in a parent/child chain. 

[1] https://github.com/torvalds/linux/blob/master/drivers/block/rbd.c#L98
[2] https://github.com/torvalds/linux/blob/master/drivers/block/rbd.c#L92

Reply:
@Madhu-1 could you  please add the links to these notes somewhere?